src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -235,6 +238,7 @@ void EnPoFusen_IncrementRomaniPop(EnPoFusen* this) {
 }
 
 void EnPoFusen_Pop(EnPoFusen* this, PlayState* play) {
+    // Spawns some smoke and some light rays for the pop.
     Actor_Spawn(&play->actorCtx, play, ACTOR_EN_CLEAR_TAG, this->actor.world.pos.x, this->actor.world.pos.y + 20.0f,
```

Comment:
Not sure this comment is needed here. You could move this comment to `ClearTagType` in `z64actor.h` though, that might be a better place for it:
`/* 0x02 */ CLEAR_TAG_POP, // Spawns some smoke and some light rays`

Reply:
Until we rename the actor, I suspect a lot of outsiders are going to be confused by what clear_tag does, m y goal here is to specify that this has NOTHING to do with "tags" or "clearing" anything, this actor is for spawning visual effects.

I don't really buy the argument that documentation to explain code in front of you should be buried in a completely different file. Anyone viewing the code on github doesn't get access to mouse over intelesense to load it from this location.

Reply:
fyi, you can press `.` on GitHub to open vsCode on browser, then searching for `CLEAR_TAG_POP` in the search will get you to `/* 0x02 */ CLEAR_TAG_POP` in 2 seconds, and alternatively because this is an actor spawn, it takes 2 seconds to `ctrl p` and typing "clear tag" to open the file. So viewing the code on github isn't a limitation imo

Reply:
There's also the disadvantage of having to duplicate this comment on every use of this enum value instead of just having it in one place, or some other use of this value not having a comment so people would need to dig for every usage of this value to find what it does.

Reply:
All I'm hearing is the code is confusing enough people have to go out of their way to figure out what this code does (or have extensive previous knowledge), and that is preferable to... a single line of comment explaining it?

Is duplicate documentation such a big problem?

Reply:
I think having a comment to explain it better is great, especially as you mention it's not immediately clear until `CLEAR_TAG` as a system is renamed. I just think it's simpler to have it in one place at the enum definition, and going to definitions of enums for clarity should be a common practice. But it's not a big deal for me either way

Reply:
It'd still be nice to have this comment on the enum, since it is helpful. But if you'd still want this comment, duplicated documentation > no documentation imo. Could always tackle it later though with more "clear-tag" docs